### PR TITLE
Add command line tool to setup database.

### PR DESCRIPTION
**Pull request by kisom**
This change introduces the initial framework (based on cobra) for setting up the trust database.

It also adds initial debian packaging metadata, for proper packaging.

### CODE REVIEW

**Comment by kisom** [comment]
Build failures look to be due to getting an older travis image repeatedly (e.g. Ubuntu 12.04); this ships with gcc 4.3, which doesn't support C++11 but does support C++0x. This is in a vendored dependency, which I'm of the mind to remove for this.

File: cli/root.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	cfgFile string
+	dbFile  string
+)
+
+func root(cmd *cobra.Command, args []string) {
```
Comment (lziest) [inline cli/root.go]: this is intentional no-op?

Comment (lziest) [inline cli/root.go]: shall we have call it noop instead?

**Comment by lziest** [comment]
LGTM in general

File: cli/root.go (hunk 1 above)
Comment (kisom) [inline cli/root.go]: For now, yes. Eventually, it'll run the help command.

**Comment by kisom** [comment]
@mihirjham also reviewed the diff, and his two main comments were:

+ Switching to transactions, and
+ Using [sqlmock](https://github.com/DATA-DOG/go-sqlmock) for testing certdb.

**Comment by kisom** [comment]
In order to have more readable diffs, though, we'll merge this PR and integrate those comments into the next one.

**Pull request merged**